src/modules/users/user_service.ts

```diff
@@ -70,34 +75,69 @@ export class UserService {
     avatar: request.avatar,
   })
 
-  private getPartnerId = async (company: string): Promise<string> => {
+  private isPartnerExist = async (company: string) => {
     const partner = await this.userRepository.findByNamePartner(company)
-    if (partner) {
-      return partner.id
+    return !!partner
+  }
```

Comment:
what is the meaning of '!!' on this return, looking for new insight 👀

Reply:
Itu shorcut dari

`result ? true : else`